### ISSUE TITLE
Enforce blank line between `package` and `import` statements

### ISSUE THREAD

**Issue by veyndan** [pinterest/ktlint#3251]
## Expected Rule behavior

When linting the following, I would expect a style violation due the lack of a blank line between the `package` and `import` statements:

```kotlin
package com.example
import foo.bar
```

## Additional information
* Current version of ktlint: 1.8.0
* Styleguide section: https://developer.android.com/kotlin/style-guide#structure

**Comment by paul-dingemans** [comment]
Sounds like a good idea. Tnx for opening the issue.

**Comment by maheen8q** [comment]
I'd like to work on this. The Android Kotlin Style Guide requires a blank line between the package statement and import statements, but ktlint currently doesn't enforce it. I'm planning to add a new rule standard:package-import-blank-line with autocorrect support. Let me know if this approach looks good before I open a PR.

**Comment by paul-dingemans** [comment]
Sure, go ahead bit should be fairly easy. I am in a kind of rush for release of 2.0 version, so please fix it soon.

Please name the rule just "package", so that it eventually also can contain other violations. If we already have a package rule, please extend that rule instead of creating a new rule.
